source/_lovelace/button.markdown

```diff
@@ -7,10 +7,35 @@ description: "The Button card allows you to add buttons to perform tasks."
 The Button card allows you to add buttons to perform tasks.
 
 <p class='img'>
-<img src='/images/lovelace/lovelace_entity_button_card.png' alt='Screenshot of the button card'>
-Screenshot of the Button card.
+<img src='/images/lovelace/lovelace_entity_button_card.png' alt='Screenshot of the button cards'>
+Screenshot of three Button Cards.
 </p>
 
+## Card Settings
+
+{% configuration_basic %}
+Entity:
+  description: The entity the card interacts with, for example `light.living_room`.
+Name:
+  description: The button name. The default is the name of the entity, if an entity is selected.
+Icon:
+  description: The icon shown on the card. The default is the entity domain icon, if an entity is selected.
+Show Name?:
+  description: A toggle to show or hide the button name.
+Show Icon?:
+  description: A toggle to show or hide the icon.
+Icon Height:
+  description: The height of the icon, in pixels.
+Theme:
+  description: The card theme, which may be set to any installed theme.
+Tap Action:
+  description: The action taken on card tap. For more information, see the [action documentation](/lovelace/actions/#tap-action).
+Hold Action:
+  description: The action taken on card tap and hold. For more information, see the [action documentation](/lovelace/actions/#hold-action).
```

Comment:
Card Tap? is this supposed to be just hold?

Reply:
I’m referencing the Actions page here, which describes it as a tap and hold. 
![F330832A-85B3-4272-8B54-12B23B95F95F](https://user-images.githubusercontent.com/64062738/88225289-55967e00-ccbe-11ea-9f2d-d712c41d4cf5.jpeg)

Reply:
Gotcha. hmm maybe that is wrong then

@bramkragten is this correct wording? It only makes sense to be only hold right?

Reply:
Tapped is weird anyway, what is a tap? A click is not a tap right? Yet it also works with a click.

But uuuuh... to hold it, you first have the tap/click it?
It is either long press or tap/touch/click & hold?

Reply:
We could move to click instead of tap I guess, but to be honest I think the meaning is pretty clear and a lot of people will have this UI set up on a touch device such as a tablet.  It also follows the naming convention in the YAML, which is `tap_action` and `hold_action`. But I do agree with @bramkragten that in order to hold, you must first tap.

Reply:
I don't think anyone will be trying to hold the button without tapping/clicking it first :)

Reply:
It's a generic name, google it 🙂

Reply:
I’ve left it as is for now. If we decide to change the Actions documentation, we can change the cards documentation to match?